assets/js/event-handler.js

```diff
@@ -30,17 +31,19 @@ document.getElementById('bug-report').addEventListener('vl-bug-report', (e) => {
       background: "#a5dc86",
       title: 'Bug Reported Successfully',
     })
-  } else {
+  } else {    
+    const error = event.detail.error;    
     Toast.fire({
       icon: 'error',
       iconColor: "white",
       color: "white",
       background: "#f27474",
       timer: 5000,
       title: 'Bug Report Failed, Please Try Again',
-    })
+      text: error ? error.message : 'An error occurred while reporting the bug'
```

Comment:
Showing the user this error message is not useful. You can log it to the console.
In most cases this message is going to be too technical for a general user to understand.